dashboard/src/components/AppView/AccessURLTable/AccessURLItem/AccessURLIngressHelper.tsx

```diff
@@ -48,6 +45,29 @@ export function GetURLItemFromIngress(ingress: IResource) {
   } as IURLItem;
 }
 
-export function IsURL(url: string): boolean {
-  return isURLRegex.test(url);
+// ShouldGenerateLink returns true if the string is, likely, a URL and
+// it, presumably, does not contain a regular expression.
+// For instance:
+//   http://wordpress.local/example-good should be true
+//   http://wordpress.local/example-bad(/|$)(.*) should be false
+export function ShouldGenerateLink(url: string): boolean {
+  // Check if it is a valid URL, delegating the check to the browser API
+  let builtURL;
+  try {
+    // Note this browser API constructor will accept many invalid URLs (not well-encoded, wrong IP octets, etc.)
+    builtURL = new URL(url);
+  } catch (_) {
+    return false;
+  }
+  if (!(builtURL.protocol === "http:" || builtURL.protocol === "https:")) {
```

Comment:
I assume we don't need to worry about `data:` urls, given they weren't matched by the previous regex.